src/style.css

```diff
@@ -1 +1,53 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo__img {
+  display: block;
+}
+
+.nav__list {
```

Comment:
Use separated css-files for block styles (BEM). Then import them to main css file (style.css)